### DISCUSSION TITLE
Ability to archive labels

### DISCUSSION THREAD

**Discussion by mnot** [community/community#5399] [Projects and Issues]
It would be *great* to be able to archive a label -- i.e., to have it still associated with issues, but not show up in the drop-down list for selecting a label to apply to an issue.

In the HTTP Working Group, we use a [repo to develop our draft specifications](https://github.com/httpwg/http-extensions/) and use labels to distinguish the issues associated with different specs. Once a spec is published, we don't raise issues on it any more, so suggesting that label for new specs isn't helpful, but we don't want to disassociate the closed issues from that label by deleting it.

A workaround is to rename the labels to sort last, but that's kind of ugly.

**Comment by jwindmuller** [comment]
This would be great and would de-clutter the UI without getting rid of valuable information.

**Comment by RSSatMonsido** [comment]
I, for one, would also like this.

**Comment by crisr15** [comment]
This would be incredibly valuable. We need the information for past projects and initiatives, but its overwhelming for new users and many are orphaned when priorities or practices change.

**Comment by harrisonokins** [comment]
This would be valuable in some projects I work on as well. We're able to accomplish similar functionality between Projects and Milestones, but some of the difficulties we run into are:

- Issues and Pull Requests can belong to multiple Projects simultaneously (which is great), but often we're tracking an arbitrary deadline or category, and consequently don't need the _full_ suite of management features Projects provide
- Milestones are almost perfect, however you can only have one assigned per Issue or Pull Request (which is by design, but in our case we have work that simultaneously belongs to multiple milestones)
- Labels allow us to arbitrarily link Issues and Pull Requests to multiple deadlines and feature work, but we've become bogged down in historical metadata that we don't want to delete, but also no longer want to use going forward

**Comment by Jon889** [comment]
+100 for this, we have a hundreds of release labels and old project labels that are no longer should be applied to PRs but are very useful to have on old PRs.

**Comment by lwld** [comment]
+1, would be very helpful. How to get this moved up in feature requests?
